package.json

```diff
@@ -38,20 +39,21 @@
     "underscore": "^1.9.1"
   },
   "devDependencies": {
-    "coveralls": "^3.0.6",
-    "nyc": "^14.1.1",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^14.0.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-import": "^2.18.2",
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
     "@babel/node": "^7.5.5",
+    "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.5.5",
     "@babel/register": "^7.5.5",
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
+    "coveralls": "^3.0.6",
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-prettier": "^6.0.0",
```

Comment:
Why did you install prettier?

Reply:
@Daymorelah I did not install prettier. I got it when I rebased my branch with the develop branch.
Here is the [package.json](https://github.com/andela/rogue-backend/blob/develop/package.json) _(line 45)_ for the develop branch.